sqlglot/dialects/clickhouse.py

```diff
@@ -51,9 +51,16 @@ class Tokenizer(tokens.Tokenizer):
             "INT32": TokenType.INT,
             "INT64": TokenType.BIGINT,
             "INT8": TokenType.TINYINT,
+            "LONGBLOB": TokenType.LONGBLOB,
+            "LONGTEXT": TokenType.LONGTEXT,
```

Comment:
they map 1-1 so let's move them to base?

Reply:
sounds good, i'll also change a couple of mappings (tinytext, tinyblob) in base to be consistent